client/guidestours/config.js

```diff
@@ -1,26 +1,52 @@
+const React = require( 'react' );
+const Gridicon = require( 'components/gridicon' );
+const i18n = require( 'lib/mixins/i18n' );
+
 const config = {
 	init: {
-		text: 'Welcome to WordPress.com!',
+		text: i18n.translate( '{{strong}}Need a hand?{{/strong}} We\'d love to show you around the place, and give you some ideas for what to do next.', {
+			components: {
+				strong: <strong />,
+			}
+		} ),
+		type: 'GuidesFirstStep',
 		next: 'my-sites',
 	},
 	'my-sites': {
 		target: 'my-sites',
-		type: 'bullseye',
+		type: 'GuidesActionStep',
+		icon: 'my-sites',
 		placement: 'below',
-		text: `First things first. Up here, you'll find tools for managing your site's content and design.`,
-		next: 'reader'
+		text: i18n.translate( "First things first. Up here, you'll find tools for managing your site's content and design." ),
+		next: 'posts-pages',
+	},
+	'posts-pages': {
+		text: i18n.translate( "Posts aren't Pages. Would you like to know more?" ),
+		type: 'GuidesLinkStep',
+		linkLabel: i18n.translate( 'Learn more about Posts and Pages' ),
+		linkUrl: 'https://en.support.wordpress.com/post-vs-page/',
+		next: 'reader',
 	},
 	reader: {
 		target: 'reader',
-		type: 'bullseye',
+		type: 'GuidesActionStep',
+		icon: 'reader',
 		placement: 'beside',
-		text: `This is the Reader. It shows you fresh posts from other sites you're following.`,
+		text: i18n.translate( "This is the Reader. It shows you fresh posts from other sites you're following." ),
```

Comment:
FWIW. Double quotes are explicitly allowed in these situations by our eslint config: https://github.com/Automattic/wp-calypso/blob/master/.eslintrc.js#L77 (`avoid-escape`)